forta-uniswap-event-py/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "uniswap-largeswap-event-agent",
+  "version": "0.0.1",
+  "description": "Detects large ETH swaps",
+  "scripts": {
+    "postinstall": "python3 -m pip install -r requirements_dev.txt",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e py --exec 'forta-agent run'",
+    "start:prod": "forta-agent run --prod",
+    "tx": "forta-agent run --tx",
+    "block": "forta-agent run --block",
+    "range": "forta-agent run --range",
+    "file": "forta-agent run --file",
+    "publish": "forta-agent publish",
+    "test": "python3 -m pytest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.0.14"
```

Consider using the most recent `forta-agent- npm package.
```suggestion
    "forta-agent": "^0.0.16"
```